x-pack/legacy/plugins/canvas/public/components/workpad_header/workpad_zoom/workpad_zoom.tsx

```diff
@@ -43,11 +55,31 @@ export class WorkpadZoom extends PureComponent<Props> {
     setZoomScale: PropTypes.func.isRequired,
     zoomIn: PropTypes.func.isRequired,
     zoomOut: PropTypes.func.isRequired,
+    resetZoom: PropTypes.func.isRequired,
+    boundingBox: PropTypes.shape({
+      left: PropTypes.number.isRequired,
+      right: PropTypes.number.isRequired,
+      top: PropTypes.number.isRequired,
+      bottom: PropTypes.number.isRequired,
+    }),
+  };
+
+  _fitToWindow = () => {
+    const { boundingBox, setZoomScale } = this.props;
+    const canvasLayoutContent = document.querySelector(
+      '.canvasLayout__stageContent'
```

Comment:
nit: I would make this an exported constant wherever it is used by the element, and import it here.